ql/src/java/org/apache/hadoop/hive/ql/exec/vector/VectorizedRowBatchCtx.java

```diff
@@ -635,4 +635,30 @@ public final boolean isPartitionCol(int colNum) {
     return colNum >= dataColumnCount && colNum < rowColumnTypeInfos.length;
   }
 
+  public final void resetVectorizedRowBatch(VectorizedRowBatch rowBatch, boolean resetPartitionColumn) {
```

Comment:
looks like resetPartitionColumn is always false

Reply:
isn't it the same as VectorizedRowBatch.reset()
````
    selectedInUse = false;
    size = 0;
    endOfFile = false;
    for (ColumnVector vc : cols) {
      if (vc != null) {
        vc.reset();
        vc.init();
      }
    }
````

Reply:
@maswin, could you please check the above comment

Reply:
@deniskuzZ : Sorry about the delay. Fixed.